TicTacToe/Game.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace TicTacToe
+{
+    public class Game
+    {
+        public Game()
+        {
+            this.Board = new string[] { "", "", "", "", "", "", "", "", "" };
```

Comment:
`this.Board = new string [9];` ⬅️ does this do the same thing?

Reply:
yes it does